src/blob_file_test.cc

```diff
@@ -88,6 +88,10 @@ class BlobFileTest : public testing::Test {
     const int n = 100;
     std::vector<BlobHandle> handles(n);
 
+    auto rate_limiter =
```

Comment:
For test, add a new test case? The test should create a mock test rate limiter, ran GC job, and check if it call into the mock rate limiter. It should test with all three values of `RateLimiter::Mode`.